spec/features/requests/personalization_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Request personalization', type: :feature do
+  let(:user) { create(:user) }
+
+  context 'given two contributors'
+  before(:each) do
+    # `broadcast!` is stubbed in tests
+    allow(Request).to receive(:broadcast!).and_call_original
+
+    create(:contributor, first_name: 'Adam', last_name: 'Apfel', email: 'adam@example.org')
+    create(:contributor, first_name: 'Zora', last_name: 'Zimmermann', email: 'zora@example.org')
+  end
+
+  scenario 'sending a request with placeholders' do
+    visit new_request_path(as: user)
+
+    fill_in 'Interner Titel', with: 'Personalizes request'
+    fill_in 'Was möchtest du wissen?', with: 'Hi {{FIRST_NAME}}, how are you?'
+
+    click_button 'Frage an die Community senden'
+
+    # TODO: This isn't really what you'd do in a feature test. However, it's an easy to
+    # understand solution, given that we don't display broadcasted messages in the UI
+    expect(Message.pluck(:text)).to contain_exactly('Hi Adam, how are you?', 'Hi Zora, how are you?')
```

Comment:
I sympathize with this cause it's actually not easy for us to run e2e tests. Maybe, we could make these contributors threema/telegram/signal contributors and stub out the external service and expect they get called with the text we expect?

That would actually be one step further down the flow.
Also, can we make this a system spec instead of a feature spec? We talked about this after you wrote this, I guess.

Reply:
> I sympathize with this cause it's actually not easy for us to run e2e tests. Maybe, we could make these contributors threema/telegram/signal contributors and stub out the external service and expect they get called with the text we expect?

Sounds sensible, I’ll look into it and see if I can get it working within a reasonable amount of time.

> Also, can we make this a system spec instead of a feature spec? We talked about this after you wrote this, I guess.

As I said previously, I’m open to migrating from feature specs to system specs. However, I’d prefer to do that consistently instead of having a mix of both scattered across different directories. Also, I think it would be good to make sure we can run them in an actual headless browser as there isn’t much of a difference apart from that, right?

Reply:
> As I said previously, I’m open to migrating from feature specs to system specs. However, I’d prefer to do that consistently instead of having a mix of both scattered across different directories.

I agree that we should be consistent, but we also have finite resources and prioritization to consider. What I've done on a few teams in the past is that when we agree that we should change some established pattern, we hold each other accountable for new code so that we don't make migrating tasks bigger in the future. We then agree to carry out a reasonable amount of refactoring when we need to touch a file anyways. The hard part is finding the time to refactor all code, but in this case it's not much.

> Also, I think it would be good to make sure we can run them in an actual headless browser as there isn’t much of a difference apart from that, right?

I have a working example of running the tests with `selenium_chrome_headless` in https://github.com/tactilenews/100eyes/pull/1412. I guess this has a higher likelihood of getting merged first, so maybe we could just run them with `rack_test` till #1412 gets merged?